v2/azure-arm.yaml

```
@@ -1561,6 +1561,51 @@ objectModelConfiguration:
         $exportAs: VirtualNetworksVirtualNetworkPeering
         $supportedFrom: v2.0.0-alpha.1
     2022-07-01:
+      ApplicationGateway:
+        $export: true
+        $supportedFrom: v2.3.0
```

I don't think this will make it for v2.3.0.

```suggestion
        $supportedFrom: v2.4.0
```

---

fixed in updated commit